pages/success-banner/index.js

```diff
@@ -1,13 +1,47 @@
 import React from 'react';
 import Head from 'next/head';
 
-class Button extends React.Component {
+const eventLog = (eventDesc) => {
+  const data = 'Banner ' + eventDesc;
+
+  return () => {
+    const url = new URL(location.href);
+
+    if (url.searchParams.get('etcher-version').indexOf('1.0.0') !== -1) {
```

Comment:
In this particular case though, we're printing in both sides of the conditional, so I guess it doesn't hurt at all to always send a robot object.